docs/get-started.asciidoc

```diff
@@ -1,23 +1,172 @@
+. After registering, click *Create deployment* at `<https://cloud.elastic.co/home>`.
+. Once that is created, click *Open* to visit your Kibana home page, e.g. `https://{DEPLOYMENT_NAME}.kb.{REGION}.cloud.es.io/app/home#/getting_started`.
+====
+
+////
+TO DO:
+Anything else they need to do/have/know?
+For example, any dependencies (including specific supported Java versions)?
+////
+
+////
+✅ How to install the distro
+////
 [discrete]
 [[get-started-install]]
 === Install
 
-. Download the latest release https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest[here].
-. Run the distro by using the `-javaagent:` JVM argument with the path to the distro jar:
+////
+✅ Step-by-step instructions
+////
+To get started with the distro:
+
+. Download the https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest[latest release].
+. Run the distro using the `-javaagent:` JVM argument with the path to the distro jar:
 +
 [source,bash]
 ----
 java -javaagent:/path/to/agent.jar \
 -jar myapp.jar
 ----
 
+////
+✅ Any additional info related to installation
+////
+[NOTE]
+.The Elastic distro and the OpenTelemetry SDK
+====
+The distro includes a transitive dependency on the OpenTelemetry SDK, so you do not _need_ to add the
+OpenTelemetry SDK package to your project directly. However, you _can_ explicitly add the OpenTelemetry SDK as a dependency if you want to opt into newer SDK versions. If you do this, the OpenTelemetry SDK dependency must
+be defined _before_ the {distro_name} is defined.
+====
+
+////
+✅ Start-to-finish operation
+////
+[discrete]
+[[get-started-send-data]]
+=== Send data to Elastic
+
+After installing the distro, configure and initialize it to start
+sending data to Elastic.
+
+////
+TO DO:
+Are there any dependencies that need to be installed in addition to the distro?
+If yes, does it matter in what order dependencies are defined?
+If no, delete this section.
+////
+[discrete]
+[[get-started-add-dependencies]]
+==== Add dependencies
+
+////
+Step-by-step instructions
+////
+
+////
+TO DO:
+What does the _minimal_ configuration / setup look like?
+Are there any configuration options that are _required_?
+////
 [discrete]
 [[get-started-configure]]
-=== Configure
+==== Configure the distro
+
+////
+Step-by-step instructions
+////
+
+////
+TO DO:
+Does the user need to do something after configuring the distro
+for data to start being sent Elastic?
+If no, delete this section.
+////
+[discrete]
+[[get-started-initialization]]
+==== Initialize the distro
+
+////
+Step-by-step instructions
+////
+
+////
+Optional: Add an image
+////
+
+////
+TO DO:
+What should they expect to see after running this?
+How do they know it worked as expected?
+////
+[discrete]
+[[get-started-view-traces]]
+=== View traces
+
+To view the traces, you can use the Elastic APM UI:
+
+////
+Step-by-step instructions
+////
+
```

Comment:
This will be generic to all agents. Basically open the APM UI and look at the services listed to find your service. We probably have this in a dozen places in the docs, so maybe we should have this centralized somewhere?